config/server/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const {
+  renderCallback,
+  middleware
+} = require('__sku_alias__serverEntry').default;
```

Comment:
Sorry if I've missed something here being late to review. But why are we destructuring a default export? Can we just use named exports instead? https://github.com/standard-things/esm Could be a good idea as well.

Reply:
I think the current approach makes sense because the interface is dynamic—`middleware` is optional right now, and there are likely to be more changes in the future.